conf/runner.yaml

```diff
@@ -64,7 +64,7 @@ runs:
         - oauth.accounts.firefox.com
         - profile.accounts.firefox.com
         - verifier.accounts.firefox.com
-        - accounts.cdn.mozilla.net
```

Comment:
This still resolves to cloudfront. Is it no longer in use, and if so, can we get rid of the record in route53?

Reply:
Yep! I just deleted the CDN, S3 bucket, and route 53 record